src/lib/config/useSupportedNetwork.ts

```diff
@@ -0,0 +1,12 @@
+import { createContext, useContext } from 'react';
```

Comment:
This seems like the correct pattern for filename and function names. But maybe something else makes more sense.

Reply:
Seems good to me.